app.rb

```diff
@@ -0,0 +1,36 @@
+require_relative './book'
+require_relative './teacher'
+require_relative './student'
+require_relative './rental'
+require_relative './person'
+require 'json'
+require_relative './user_input'
+require_relative './user_output'
+
+class App
+  def initialize()
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def run
+    UserOutput.load_people(@people)
+    UserOutput.load_books(@books)
+    UserOutput.load_rentals(@rentals)
+    user_input = 0
+    puts "####\nWelcome to the School Library Ruby App!\n####"
+    while user_input != '7'
+      puts "Choose a number from the options below:\n"
+      app_options
+      print 'Enter Option [number]: '
+      user_input = gets.chomp
+      puts "\n\n"
+      check_selection(user_input)
+    end
+    puts "Exiting, thanks for using this app!\n\n" if user_input == '7'
```

Comment:
- [ ] Your app is throwing an error in an attempt to exit, using option 7, as can be seen in the screenshot below.
Kindly ensure to improve upon this aspect.

**Exit error:**
![image](https://user-images.githubusercontent.com/12745474/202795826-ffbf2eea-9a64-4dec-82b0-bbca4018a538.png)

Reply:
As for this, I corrected it before requesting for review, I don't know why it is still showing

I will check again

Reply:
Fixed